auth-web/src/components/auth/account-settings/team-management/InvitationsDataTable.vue

```diff
@@ -95,6 +103,10 @@ export default class InvitationsDataTable extends Vue {
     }
   ]
 
+  public mounted () {
+    this.canDoActions = !this.currentUser?.roles?.includes(Role.ContactCentreStaff)
```

Comment:
Would it be better to just abstract this a bit since we are re-using v-can and the common utils Permissions?
We can add a permission there and not need to add the role check here specifically?